modules/test/playwright/tests/portal-workflow-task-web/assignments.spec.ts

```diff
@@ -151,3 +161,149 @@ test('send user back to my workflow tasks page after assign another user to revi
 
 	await expect(workflowTasksPage.assignedToMyRolesLink).toBeVisible();
 });
+
+test('logged user must be able to see workflow task at least from a read-only perspective', async ({
+	apiHelpers,
+	configurationTabPage,
+	diagramViewPage,
+	messageBoardsEditThreadPage,
+	messageBoardsPage,
+	messageBoardsWidgetPage,
+	page,
+	processBuilderPage,
+	site,
+	userPersonalBarPage,
+	workflowTaskDetailsPage,
+	workflowTasksPage,
+}) => {
+	const user =
+		await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+			'demo.unprivileged@liferay.com'
+		);
+
+	const defaultUser =
+		await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+			'test@liferay.com'
+		);
+
+	const content = JSON.parse(
```

Comment:
Hey Richard, I believe we can name this variable with a more suggestive name, perhaps specifying the type of content we are assigning.